ci/long_running_tests/workloads/many_drivers.py

```diff
@@ -8,7 +8,7 @@
 
 import ray
 from ray.tests.cluster_utils import Cluster
-from ray.tests.utils import run_string_as_driver
+from ray.test_utils import run_string_as_driver
```

Comment:
The tests packages is now removed, since the conftest.py import was causing import mismatch errors between the locally installed test dir and the copy of the sources used by bazel to run the test.

Reply:
This seems to be a somewhat fundamental issue with the way we are using py_test and the local install.

The only reason pytest worked before is because it was running files in the local install and not the source tree, so there was no conflict.